package.json

```diff
@@ -46,32 +45,34 @@
     "algorand",
     "ripple",
     "evm",
-    "ens"
+    "ens",
+    "ican"
   ],
   "author": "Sergiy Dybskiy",
   "license": "MIT",
   "dependencies": {
-    "bs58check": "^3.0.1",
-    "ethereum-cryptography": "^2.1.2"
+    "bs58check": "^4.0.0",
+    "ethereum-cryptography": "^3.0.0"
   },
   "devDependencies": {
-    "@size-limit/esbuild": "^11.0.1",
-    "@size-limit/esbuild-why": "^11.0.1",
-    "@size-limit/preset-small-lib": "^11.0.1",
-    "@types/node": "^20.10.0",
-    "@typescript-eslint/eslint-plugin": "^6.13.0",
-    "@typescript-eslint/parser": "^6.13.0",
-    "@vitest/coverage-v8": "^1.0.0",
-    "eslint": "^8.54.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-prettier": "^5.0.1",
-    "husky": "^8.0.3",
-    "lint-staged": "^15.1.0",
-    "prettier": "^3.1.0",
-    "size-limit": "^11.0.1",
-    "tsup": "^8.0.0",
-    "typescript": "^5.3.0",
-    "vitest": "^1.0.0"
+    "@size-limit/esbuild": "^11.1.6",
+    "@size-limit/esbuild-why": "^11.1.6",
+    "@size-limit/preset-small-lib": "^11.1.6",
+    "@types/node": "^22.10.2",
+    "@typescript-eslint/eslint-plugin": "^8.18.2",
+    "@typescript-eslint/parser": "^8.18.2",
+    "@vitest/coverage-v8": "^2.1.8",
+    "eslint": "^9.17.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "globals": "^15.14.0",
```

Comment:
what's the reason for this addition?

Reply:
Simply required byt v9. If we want to stay up to date